pyproject.toml

```diff
@@ -1,33 +1,50 @@
 [tool.towncrier]
     package = "tox"
-    package_dir = ".."
     filename = "CHANGELOG.rst"
-    issue_format = "`#{issue} <https://github.com/tox-dev/tox/issues/{issue}>`_"
+    directory = "tox/changelog"
+    template = "tox/changelog/template.jinja2"
+    title_format = "{version} ({project_date})"
+    underlines = ["=", "-"]
 
     [[tool.towncrier.section]]
         path = ""
 
-    [[tool.towncrier.type]]
-        directory = "feature"
-        name = "Features"
-        showcontent = true
+    [[tool.towncrier.format]]
+        kind = "issue"
+        patterns = ["issue(\\d)", "(\\d)"]
+        format = "`#{id} <https://github.com/tox-dev/tox/issues/{id}>`_"
+
+    [[tool.towncrier.format]]
+        kind = "pull request"
+        patterns = ["pr(\\d)", "pull-request(\\d)"]
+        format = "`#{id} <https://github.com/tox-dev/tox/pull/{id}>`_"
```

Comment:
FYI GitHub doesn't care if you use `/pull/` or `/issue/` on the URL as long as the number is correct.

Reply:
Really? Can't they overlap? Why did nobody tell me earlier? All this programming for nothing? I am disappoint :(

Reply:
oh dear ... anyway. I had a blissfull afternoon exploring towncrier and all, so who cares :)

Reply:
I think it's a complete UX fail then that they even have /issues and pull/ - they should just have entity/ or sometting and don't redirect from issue to pull when you ref a pull from an issue url.

Reply:
Yep, it is weird 😁

Reply:
FYI, although it is a redirect, it's a permanent redirect; so if you leave it as it is the sphinx link checker would complain about it.

Reply:
Thanks @gaborbernat - I will keep that in mind when finishing up today.